ztp/gitops-subscriptions/ACMPolicyGeneratorIntergration.md

```diff
@@ -0,0 +1,183 @@
+## Generating policies using ACM Policy Generator and GitOps
+
+[ZTP container](../resource-generator/README.md) provides a list of default [source CRs](../source-crs/README.md) to distribute the RAN configurations via ACM policies generated by [ZTP Policy Generator](../policygenerator/README.md). If you have configuration CRs or validation CRs to be applied to managed clusters but are not provided in the ZTP container, you could use the [ACM Policy Generator](https://github.com/stolostron/policy-generator-plugin) to help building your custom policies.
+
+### Building custom validator policy using ACM Policy Generator
+This example describes how to generate a custom validator policy that validates the extra resources within the ZTP workflow.
+
+* Create the source files with the CRs you want to validate in a directory `custom_source_crs/validationCRs/`. The following example checks a `LocalVolume` available status:
+```
+apiVersion: local.storage.openshift.io/v1
+kind: LocalVolume
+metadata:
+  name: fs
+  namespace: openshift-local-storage
+status:
+  conditions:
+  - status: "True"
+    type: Available
+```
+Note: the validation CR can only contain `status`.
+
+* Create a `PlacementRule` manifest called `validator-placementrule.yaml`.
+```
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: extended-validator-placementrules
+    namespace: ztp-group
+spec:
+    clusterSelector:
+        matchExpressions:
+            - key: group-du-sno
+              operator: Exists
+            - key: ztp-done
+              operator: DoesNotExist
+```
+
+* Create a ACM Policy Generator configuration file called `extended-du-validator.yaml`
+```
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: extended-du-validator
+policyDefaults:
+  policyAnnotations: {"ran.openshift.io/ztp-deploy-wave": "10000"}
+  evaluationInterval:
+    compliant: 24h
+    noncompliant: 10s
+  namespace: "ztp-group"
+  placement:
+    placementRulePath: validator-placementrule.yaml
+  remediationAction: inform
+  complianceType: musthave
+policies:
+- name: "extended-du-validator-policy"
+  manifests:
+    - path: custom_source_crs/validationCRs/
+```
+
+The annotation "ran.openshift.io/ztp-deploy-wave" is required to be added in the policy for [TALM](https://github.com/openshift-kni/cluster-group-upgrades-operator) to know the order of policies to be applied/checked. This can be done via `policyDefaults.policyAnnotations` as shown above.
+
+* Add `extended-du-validator.yaml` in the `kustomization.yaml` file as a generator.
+
+
+See [ACM PolicyGenerator reference YAML](https://github.com/stolostron/policy-generator-plugin/blob/main/docs/policygenerator-reference.yaml) for the full structure of `PolicyGenerator`. 
+
+### Integrating with Openshift GitOps(ArgoCD)
```

Comment:
This section and the following section detail two different methods of integrating ACM PolicyGenerator into the GitOps ZTP flow that we define. I think it would be helpful to put both into one section with a short introduction paragraph that explains that there are two methods, both will work equivalently, and if there is a recommendation of which one is preferred we should give some rationale for that recommendation.